depclean-core/src/main/java/se/kth/depclean/core/analysis/asm/ConstantPoolParser.java

```diff
@@ -100,51 +100,29 @@ static Set<String> parseConstantPoolClassReferences(ByteBuffer buf) {
     for (int ix = 1, num = buf.getChar(); ix < num; ix++) {
       byte tag = buf.get();
       switch (tag) {
-        case CONSTANT_UTF8:
-          stringConstants.put(ix, decodeString(buf));
-          continue;
```

Comment:
@cesarsotovalero Since there's no other statement after the switch block inside the for loop, "continue" behaves same as "break" in this case.